crates/graphql/src/main.rs

```diff
@@ -75,7 +73,7 @@ fn main() {
 
         let graphiql_uri = format!("http://{}", addr);
 
-        let schema = Arc::new(schema::create(db_pool.clone()));
+        let schema = Arc::new(schema::create());
```

Comment:
Schema prob doesn't need to be an arc now.

Reply:
@kespinola that may be a juniper thing, although it'd be neat if we could drop the arc

Reply:
Can try in a later PR.